web-ui/webpack.prod.js

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
Builds `docs/index.html` which we'll hook gh-pages up to. The JS and CSS are inlined into the resulting index.html file as well - its all about 48k (most of the weight coming from `json-to-simple-graphql-schema` itself.